### PR TITLE
Analytics package

### PR DESCRIPTION
**Pull request by hyperh**
Tracking user sign in, sign out, url changes, and collection hooks.

### CODE REVIEW

**Comment by adrianmc** [comment]
I actually originally intended this repo to literally be just an example repo of the Meteor React tutorial. I was going to suggest we make a separate repo with Matt's private account to develop our thing. Nevertheless, I guess we'll just go with this for now? Thoughts? @hyperh @chunsli

File: packages/analytics/client/meteor-analytics.js [hunk 1]
```diff
@@ -147,6 +147,11 @@ function makePage(context) {
   return page;
 }
 
+function handleBeforeUnload (e) {
```
Comment (adrianmc) [inline packages/analytics/client/meteor-analytics.js]: @hyperh Awesome! Has this been tested to work already? Or did you just put in the function for now?

Comment (hyperh) [inline packages/analytics/client/meteor-analytics.js]: @adrianmc It works. Tested to work for back, refresh, tab close, window close.

**Comment by hyperh** [comment]
@adrianmc Yes, should hand ownership to @chunli.

**Pull request merged**